### PR TITLE
Add Index and Calculating Emissions, set up scss files

### PR DESCRIPTION
**Pull request by chrismekim**
Figma: https://www.figma.com/file/bWWgCqrjStjy1ne99Vs6x9/Sanctuary-Sustainability?node-id=457%3A166

Note: This doesn't include the landing with the menu and globe (designs are in progress).

### CODE REVIEW

**Comment by brendonsanctuary** [comment]
**Across the whole site**

Desktop and Mobile:
Change image captions margin top to .5rem

margin-top:.5rem;

**Comment by brendonsanctuary** [comment]
Mobile type sizes:

Screen Shot 2020-07-08 at 11 42 07 AM
Screen Shot 2020-07-08 at 11 42 00 AM
Screen Shot 2020-07-08 at 11 41 55 AM

**Comment by brendonsanctuary** [comment]
Button sizes on desktop got smaller

Screen Shot 2020-07-08 at 1 02 46 PM

**Comment by brendonsanctuary** [comment]
Let's change the intro back to the Subhead type size, but align it on this grid. The footer can still be the larger text size.
Screen Shot 2020-07-08 at 1 11 39 PM

**Comment by brendonsanctuary** [comment]
Update image layouts on the homepage. If we could just stack the images in this type of grid. I would just export the images from the figma at 1.5x for now so we can get the right sizing.

**Comment by brendonsanctuary** [comment]
DESKTOP: Spacing
on all pages lets make the spacing between sections.

15rem

**Comment by brendonsanctuary** [comment]
Desktop Carbon Emissions:
Let's move this image over a little bit.

Screen Shot 2020-07-08 at 2 26 40 PM

Screen Shot 2020-07-08 at 2 26 32 PM

**Comment by brendonsanctuary** [comment]
Footer on hover fade to gray. All links.

Screen Shot 2020-07-08 at 2 41 50 PM

**Comment by brendonsanctuary** [comment]
Desktop & Mobile SVG Padding.

Screen Shot 2020-07-08 at 2 48 31 PM
Screen Shot 2020-07-08 at 2 48 38 PM

**Comment by brendonsanctuary** [comment]
Can we move make the column 6/6 for the intro. 
Screen Shot 2020-07-08 at 5 18 07 PM

**Comment by brendonsanctuary** [comment]
Can we change the section spacing from 15REM to 12REM.

**Comment by brendonsanctuary** [comment]
12 Rem spacing between the header globe block and the intro text. 
Screen Shot 2020-07-08 at 5 45 03 PM

**Comment by brendonsanctuary** [comment]
Just changing some of the rems for the images.

Screen Shot 2020-07-08 at 5 57 15 PM
Screen Shot 2020-07-08 at 5 57 07 PM

**Comment by brendonsanctuary** [comment]
Grid change on the 4th section of images. 

Screen Shot 2020-07-08 at 6 06 32 PM

**Comment by brendonsanctuary** [comment]
Add a fade on hover over for footer links. Timing can be same as buttons.

**Comment by brendonsanctuary** [comment]
Image section margins


Screen Shot 2020-07-08 at 6 43 27 PM
Screen Shot 2020-07-08 at 6 43 50 PM
Screen Shot 2020-07-08 at 6 52 58 PM

**Comment by brendonsanctuary** [comment]
Update SVG of GHG Protocol with the one i sent you in twist.

**Comment by brendonsanctuary** [comment]
Screen Shot 2020-07-09 at 10 47 24 AM
Remove this margin top above this graphic.

**Comment by brendonsanctuary** [comment]
Screen Shot 2020-07-09 at 10 52 00 AM


Make all active links italic in the footer.
On the homepage this would mean that the "Studio Carbon Negative" link in the footer should be gray italic.

**Comment by brendonsanctuary** [comment]
Captions across the site should all be the same width. 
Screen Shot 2020-07-09 at 10 53 45 AM

**Comment by brendonsanctuary** [comment]
Desktop Caption type sizes
Screen Shot 2020-07-09 at 11 57 38 AM


Mobile Caption type sizes

Screen Shot 2020-07-09 at 11 57 41 AM

**Comment by brendonsanctuary** [comment]
Mobile Index Header: Just trying to make it a little more impactful and centered to the 100vh intro.

Remove margin-bottom from header.
Screen Shot 2020-07-09 at 12 07 18 PM

Make the globe larger
Screen Shot 2020-07-09 at 12 06 55 PM

More compelling composition:
Screen Shot 2020-07-09 at 12 12 49 PM

**Comment by brendonsanctuary** [comment]
Mobile button padding:
Screen Shot 2020-07-09 at 12 14 36 PM

**Comment by brendonsanctuary** [comment]
Mobile List:

Margin to 2.5rem
Screen Shot 2020-07-09 at 12 16 25 PM

**Comment by brendonsanctuary** [comment]
Mobile Type Updates:

Paragraph text .8rem with line-height of 1.3rem
Screen Shot 2020-07-09 at 12 34 32 PM

Make the button text and small text .66 rem
Screen Shot 2020-07-09 at 12 35 48 PM
Screen Shot 2020-07-09 at 12 34 25 PM

**Comment by brendonsanctuary** [comment]
Mobile footer:
Change lineheight to .66rem

Screen Shot 2020-07-09 at 12 37 43 PM

**Comment by brendonsanctuary** [comment]
Emissions header mobile:

Caption ( this should go across all other page headers on mobile as well)
_Moving it down a little bit_
Screen Shot 2020-07-09 at 1 05 59 PM


Moving "Emissions" Down
Screen Shot 2020-07-09 at 1 05 54 PM

Moving image down slightly as well
Screen Shot 2020-07-09 at 1 05 49 PM

**Comment by brendonsanctuary** [comment]
MOBILE:
Adding margins to the top of charts.

2 rem top margins 

Screen Shot 2020-07-09 at 1 08 42 PM
Screen Shot 2020-07-09 at 1 09 04 PM

**Comment by brendonsanctuary** [comment]
Footer text Mobile:

Screen Shot 2020-07-09 at 1 14 54 PM

**Comment by brendonsanctuary** [comment]
Mobile Sanctuary Logo:
62 px
Screen Shot 2020-07-09 at 1 17 54 PM

**Comment by brendonsanctuary** [comment]
Header home page

Screen Shot 2020-07-09 at 1 30 45 PM

**Comment by brendonsanctuary** [comment]
Desktop:
Removing this 10rem margin here. 
Screen Shot 2020-07-09 at 1 35 44 PM

**Comment by brendonsanctuary** [comment]
Desktop & Mobile Scope Titles on Carbon Emissions:
Screen Shot 2020-07-09 at 1 39 16 PM

Can we change the "Scopes" and 0% to these type styles.

Desktop
Screen Shot 2020-07-09 at 1 39 05 PM

Mobile
Screen Shot 2020-07-09 at 1 39 06 PM

**Comment by brendonsanctuary** [comment]
Mobile:
Remove header margin

Screen Shot 2020-07-09 at 1 54 22 PM

**Comment by brendonsanctuary** [comment]
Pushing the globe down a little and the image down a little.

Globe
Screen Shot 2020-07-09 at 1 58 18 PM

Image
Screen Shot 2020-07-09 at 1 58 08 PM

**Comment by brendonsanctuary** [comment]
I think the size of this image

Feels cool and we should have this size be the same over the globe.
Screen Shot 2020-07-09 at 2 11 02 PM


We should have the image be Col5 like the Emissions page.
I change the Globe to be 90vw.
I moved the image top 60%.
Screen Shot 2020-07-09 at 2 15 37 PM

Screen Shot 2020-07-09 at 2 14 20 PM

**Comment by brendonsanctuary** [comment]
Logo on homepage needs to be 62px to match emissions page. 

Screen Shot 2020-07-09 at 2 18 01 PM

**Comment by brendonsanctuary** [comment]
Mobile:
Anytime there is a button the margin-top should be 1 rem :)

Screen Shot 2020-07-09 at 2 57 48 PM

**Comment by brendonsanctuary** [comment]
Woops it looks like the desktop globe got uncentered. 
 

Screen Shot 2020-07-09 at 3 03 16 PM

**Comment by brendonsanctuary** [comment]
Desktop Globe Centering & Size:

Screen Shot 2020-07-09 at 4 05 57 PM

Desktop:
Content Margin: 6 rem
Screen Shot 2020-07-09 at 4 07 39 PM

File: src/css/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+.text-footer-xs {
+  font-family: $atlas;
+  font-size: 0.55rem;
+  line-height: 0.77rem;
+
+  @include media('md-up') {
+    font-size: 0.77rem;
+    line-height: 1.11rem;
+  }
+}
```
Comment (mokaymokay) [inline src/css/_typography.scss:139]: these classes probably shouldn't have component-specific names, makes it harder to re-use

File: src/css/main.scss [hunk 2]
```diff
@@ -40,77 +37,326 @@ header {
...
-  margin: auto;
-  bottom: 0;
+.header-image-container {
+  top: 60%;
+  left: 38%;
+  max-height: 8.61rem;
+  transform: translate(-50%, -50%);
+
+  > img {
+    object-fit: contain;
```
Comment (mokaymokay) [inline src/css/main.scss:163]: basement has `.fit-contain`!

File: src/css/main.scss [hunk 3]
```diff
@@ -40,77 +37,326 @@ header {
...
-  font-size: 12px;
-  background-color: transparent;
-  border-width: 1px;
-  border-color: black;
-  border-style: solid;
-  border-radius: 999px;
+//Calculating Emissions
+.calculating {
+  margin-top: 10vh;
+  z-index: 1;
```
Comment (mokaymokay) [inline src/css/main.scss:196]: use `.z1` from basement

File: src/css/main.scss [hunk 4]
```diff
@@ -40,77 +37,326 @@ header {
...
+    margin-top: 50vh;
+  }
 }
 
-footer {
-  img {
-    max-width: 80px;
+.emissions {
+  margin-top: 60vh;
+  z-index: 1;
```
Comment (mokaymokay) [inline src/css/main.scss:213]: use .z1 from basement

File: src/css/main.scss [hunk 5]
```diff
@@ -40,77 +37,326 @@ header {
...
+  padding-bottom: 0.55rem;
+  border-top: 1px solid color('black');
+}
+
+//Footer
+
+.footer-logo-container > img {
+
+  @include media('sm-down') {
+    max-width: 62px;
```
Comment (mokaymokay) [inline src/css/main.scss:319]: use rem

File: src/css/main.scss [hunk 6]
```diff
@@ -40,77 +37,326 @@ header {
...
+}
+
+.emissions-header-image-container {
+  top: 23%;
+  left: 6%;
+  left: 1.112rem;
+  max-height: 20rem;
+
+  > img {
+    object-fit: cover;
```
Comment (mokaymokay) [inline src/css/main.scss:236]: `.fit-cover`

File: src/css/main.scss [hunk 7]
```diff
@@ -40,77 +37,326 @@ header {
   }
 }
 
-.wrapper {
-  max-width: 1280px;
-  margin: auto;
+.header {
+  height: 100vh;
```
Comment (mokaymokay) [inline src/css/main.scss:41]: `.vh100`

File: src/css/main.scss [hunk 8]
```diff
@@ -40,77 +37,326 @@ header {
...
 }
 
-h1 {
-  font-size: 70px;
+//Index
+
+.map-container {
+  position: relative;
+  width: 90vw;
+  overflow: hidden;
```
Comment (mokaymokay) [inline src/css/main.scss:128]: `.relative` and `.overflow-hidden`

File: src/css/main.scss [hunk 9]
```diff
@@ -40,77 +37,307 @@ header {
...
+  margin-left: 1.11rem;
+  margin-right: 1.11rem;
 }
 
-h1, h2, h3, h4, h5, h6, p.serif {
-  font-weight: 100;
+.content-container {
+  display: flex;
+  margin-bottom: 5rem;
+  max-width: 1440px;
```
Comment (limsohee1002) [inline src/css/main.scss:49]: Why this one using px value? Any specific reason?

Comment (chrismekim) [reply src/css/main.scss]: No particular reason, I changed to rem! :)

**Pull request merged**